.prettierrc

```diff
@@ -1,3 +1,4 @@
 semi: true
 trailingComma: "es5"
 singleQuote: true
+arrowParens: "avoid"
```

Comment:
Prettier was updated in 4.2.0 and that update changed the default value of `arrowParens`. This just resets that default back to what it was prior to the Prettier update.